.github/workflows/ci.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.8', '3.11']
-        toxenv: [quality, docs, pii_check, django42-celery53]
+        python-version: ['3.11', '3.12']
+        toxenv: [quality, docs, pii_check, django42-celery53, django52-celery53]
```

Comment:
The celery version is being upgraded to `5.5.x` in the requirement files in this PR.

Reply:
Updated all references to point towards latest `celery==5.5.*` dependency.